web/css/magento/block.less

```diff
@@ -0,0 +1,361 @@
+// *****************************************************
+//  * @package Venus Theme Framework for Magento 2.0.x
+//  * @version 1.0
+//  * @author http://www.venustheme.com
+//  * @copyright Copyright (C) Augus 2015 VenusThemes.com <@email:venusthemes@gmail.com>.All rights reserved.
+//  * @license   GNU General Public License version 1
+// ******************************************************
+.block {
+	.block-title {
+		text-align: left;
+		font-size: 24px;
+		font-family: 'Open Sans', sans-serif;
+		position: relative;
+	}
+	.block-content{
+		padding: 15px 0;
+	}
+
+	&-static{
```

Comment:
I never knew this "parent selector" thing existed in LESS...  There is a place I can use it in the research frontend.